Makefile

```diff
@@ -122,7 +122,9 @@ install-precommit-hook: install-pre-commit
 	pre-commit install
 
 test:  ## run backend tests
-	$(call run_engine_docker_command,pytest)
+# always use settings.ci-test django settings file when running the tests
+# if we use settings.dev it's very possible that some fail just based on the settings alone
+	$(call run_engine_docker_command,pytest --ds=settings.ci-test)
```

Comment:
without this ~46 tests fail locally just because of the settings file used when running the tests